### PR TITLE
stated intent. i will be getting rid of, commenting, and documenting …

### PR DESCRIPTION
**Pull request by bobalobdob**
…code in preferences.dm. may god rest my soul.

simply going to fix grammar and spelling, as well as refactoring some of the code and getting rid of what we dont need.

### CODE REVIEW

**Comment by bobalobdob** [comment]
WIP

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -51,7 +55,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	var/real_name						//our character's name
 	var/be_random_name = 0				//whether we'll have a random name every round
 	var/be_random_body = 0				//whether we'll have a random body every round
-	var/gender = MALE					//gender of character (well duh)
+	var/gender = MALE/*MALE || FEMALE*/ //gender of character (well duh)
```
Comment (Light-Weave) [inline code/modules/client/preferences.dm]: See if we can add nonbinary while you're here? Pronoun support already exists in DM, might require a bit of poking though.

Comment (bobalobdob) [reply code/modules/client/preferences.dm]: sure, although doesnt make much sense game-wise. doable though. will have to find where male and female are created, then copy/paste a third constructor.

**Comment by bobalobdob** [comment]
finished part one so that light-weave can add more to the file. will resume from line 150 when their changes are merged and i pull them.

**Pull request merged**